### PR TITLE
feat(docker): support centos6 build env 

### PR DESCRIPTION
**Pull request by foreverneverer**
# Ref-issue
https://github.com/apache/incubator-pegasus/issues/903

### CODE REVIEW

**Review by acelyc111 — APPROVED** [review]

**Review by acelyc111 — CHANGES_REQUESTED** [review]
wait plz

**Comment by foreverneverer** [comment]
@acelyc111 I have fixed it and now can review it

File: docker/pegasus-build-env/centos6/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# use the centos with valid yum repo url
+FROM apachepegasus/centos-vault:6.10
```
Comment (acelyc111) [inline docker/pegasus-build-env/centos6/Dockerfile:19]: What's the different between this image and `centos:centos6.10`?

File: docker/pegasus-build-env/centos6/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+libaio-devel \
+snappy-devel \
+bzip2-devel \
+zlib \
+zlib-devel \
+libzstd-devel \
+lz4-devel \
+bison \
+flex \
+patch; \
```
Comment (acelyc111) [inline docker/pegasus-build-env/centos6/Dockerfile:56]: Can krb5-devel and cyrus-sasl-devel be installed on this platform?

File: docker/pegasus-build-env/centos6/Dockerfile (hunk 1 above)
Comment (foreverneverer) [reply docker/pegasus-build-env/centos6/Dockerfile:19]: apachepegasus/centos-vault:6.10 use valid repo url

File: docker/pegasus-build-env/centos6/Dockerfile (hunk 2 above)
Comment (foreverneverer) [reply docker/pegasus-build-env/centos6/Dockerfile:56]: Yes, but its version is low and can't be used for pegasus

**Review by acelyc111 — APPROVED** [review]
Good job!

**Review by hycdong — APPROVED** [review]

**Pull request merged**